src/teb_local_planner_ros.cpp

```diff
@@ -374,10 +376,14 @@ bool TebLocalPlannerROS::computeVelocityCommands(geometry_msgs::Twist& cmd_vel)
     last_cmd_ = cmd_vel;
     return false;
   }
+  double max_vel_x =  cfg_.robot.max_vel_x;
+  double max_vel_theta = cfg_.robot.max_vel_theta;
+  speed_limit_manager_.setPlan(transformed_plan);
+  bool status = speed_limit_manager_.calculateLimits(max_vel_x, max_vel_theta);
```

Comment:
do we do anything with the status variable? should we put it in log debug or something?

Reply:
We don't use it at all, it just returns false when a limiter fails. I'll put it in the debug log